client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -36,14 +42,7 @@ describe( 'EditGravatar', () => {
 		};
 	} );
 
-	beforeAll( function () {
-		EditGravatar = require( 'calypso/blocks/edit-gravatar' ).EditGravatar;
-		FilePicker = require( 'calypso/components/file-picker' );
-		Gravatar = require( 'calypso/components/gravatar' ).default;
-		ImageEditor = require( 'calypso/blocks/image-editor' );
-		VerifyEmailDialog = require( 'calypso/components/email-verification/email-verification-dialog' );
-		DropZone = require( 'calypso/components/drop-zone' ).default;
-	} );
+	beforeAll( function () {} );
```

Comment:
This empty `beforeAll` could be removed for good?

Reply:
Good catch, removed